### PR TITLE
PT-156734655: Add many nodes stay in sync test

### PR DESCRIPTION
**Pull request by eproxus**
Simple test that mines 500 more blocks on top of the already 10 000 long chain.

### CODE REVIEW

File: system_test/aest_perf_SUITE.erl [hunk 1]
```diff
@@ -138,6 +136,20 @@ sync_speed(Cfg) ->
...
+    setup_nodes(cluster(Nodes, #{
+        mine_rate => ?cfg(mine_rate),
+        source => ?cfg(source)
+    }), Cfg),
+    [start_node(N, Cfg) || N <- Nodes],
+    wait_for_startup(Nodes, Height, Cfg),
+    wait_for_value({height, Height + 500}, Nodes, ?cfg(mine_timeout) * 500, Cfg),
+
+    Blocks = [get_block(N, Height + 500, Cfg) || N <- Nodes],
+    [?assertEqual(A, B) || A <- Blocks, B <- Blocks, A =/= B].
```
Comment (lucafavatella) [inline system_test/aest_perf_SUITE.erl]: Not sure how the `A =/= B` filter plays with `?assertEqual(A, B)`. Maybe you meant to keep the node in the structure, and filter only couples of distinct nodes?

Comment (eproxus) [reply system_test/aest_perf_SUITE.erl]: Agree, that makes more sense. And makes the assert error better.

**Comment by lucafavatella** [comment]
CI job test [failed]https://circleci.com/gh/aeternity/epoch/11683?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) with socket_closed_remotely unrelated to this PR. Restarting job.

**Comment by lucafavatella** [comment]
I understand this delivers https://www.pivotaltracker.com/story/show/156734655

**Pull request merged**